docs/methodology.md

```diff
@@ -1,566 +1,166 @@
-
-schema_config:
-  configs:
-    - from: 1972-10-24
-      store: tsdb
-      object_store: filesystem
-      schema: v13
-      index:
-        prefix: index_
-        period: 24h
-
-limits_config:
-  reject_old_samples: false
-  ingestion_rate_mb: 1000
-
-ruler:
-  alertmanager_url: http://localhost:9093
-```
-
-Then launch a container for Loki with the following command (assuming you are currently at the
-directory which contains the `yaml` configuration file above):
-```shell
-docker run \
-  --name loki \
-  -d \
-  -v $(pwd):/mnt/config \
-  -p 3100:3100 \
-  grafana/loki:3.0.0 \
-  -config.file=/mnt/config/loki-config.yaml
-```
-
-To run Promtail, create a `promtail-config.yaml` file as the following:
-```yaml
-server:
-  http_listen_port: 9080
-  grpc_listen_port: 0
-
-positions:
-  filename: /tmp/positions.yaml
-
-clients:
-  - url: http://loki:3100/loki/api/v1/push
-
-scrape_configs:
-- job_name: system
-  static_configs:
-  - targets:
-      - localhost
-    labels:
-      job: benchlogs
-      __path__: /mnt/datasets/hadoop/worker*/*
-```
-
-Note that `__path__` should be the pattern of directories which contain the log files.
-
-Then launch a container for Promtail with the following command (assuming you are currently at the
-directory which contains the above mentioned `yaml` configuration file):
-```shell
-docker run \
-  --name promtail \
-  -d \
-  -v $(pwd):/mnt/config \
-  -v /path/to/hadoop-log-datasets:/mnt/datasets/hadoop \
-  --link loki \
-  grafana/promtail:3.0.0 \
-  -config.file=/mnt/config/promtail-config.yaml
-```
-
-When containers for Loki and Promtail have been launched, run the following command until it prints
-`ready`, then Loki will start ingesting data:
-```shell
-curl -G http://localhost:3100/ready
-```
-
-#### Ingest
-Loki ingests data automatically when connects to Promtail. We do not do any preprocessing for the
-dataset.
-
-We use `docker stats` to get the memory usage of ingesting data periodically (the frequency should
-be consistent with the `yaml` configuration file for `clp-bench as mentioned in the next section)
-until the ingestion finishes:
-```shell
-docker stats loki promtail --no-stream
-```
 
-Since Loki does not have prompt when ingestion finishes, we monitor the current ingested data by the
-following command:
-```shell
-curl -G http://localhost:3100/metrics | grep 'loki_distributor_bytes_received_total'
-```
+The benchmark currently tests the following tools:
 
-When the above command prints the size that equals to the size of the dataset, it means the
-ingestion finishes. Then we run the following command to get the time spent for ingesting data:
-```shell
-curl -G http://localhost:3100/metrics | \
-  grep 'loki_request_duration_seconds_sum{method="POST",route="loki_api_v1_push"'
-```
+- For unstructured logs:
+  - [CLP][glt] (📐[methodology][glt-methodology]). Specifically, the `glt` binary.
+  - [Elasticsearch] (📐[methodology][Elasticsearch-unstructured-methodology]).
```

Comment:
Link is broken/missing.

Reply:
Which links do you mean? ([Elasticsearch] itself has been defined as a hyperlink at the bottom.)

Reply:
Sorry, I meant to comment on the Elasticsearch semi-structured one.